examples/cli_agent.py

```
@@ -1,25 +1,24 @@
 
 def main(
-    task: str = typer.Option(..., help="Task to perform"),
-    reasoning_model: str = typer.Option(LlmModel.default, help="Reasoning model to use"),
-    headless: bool = typer.Option(False, help="Run in headless mode"),
-):
-    reasoning_model = LlmModel(reasoning_model) if isinstance(reasoning_model, str) else reasoning_model.default
-    headless = headless if isinstance(headless, bool) else headless.default
-    agent = Agent(headless=headless, reasoning_model=reasoning_model)
-    return agent.run(task)
+    task: Annotated[str, typer.Option(..., help="Task to perform")],
+    reasoning_model: Annotated[str, typer.Option(help="Reasoning model to use")] = LlmModel.default(),  # type: ignore[reportArgumentType]
```

Pylint (3.3.7)

[convention] Line too long (138/100)

(C0301)